manifests/spoke/spoke-clusterrolebinding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: system:open-cluster-management:{{ .SpokeCoreName }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Let's start smaller.

Can we start with `admin` across all namespaces?  Then add a comment indicating that we want to develop a controller that could work with permissions below cluster-admin.  the tradeoff is responsivity because list/watch cannot be maintained over too many namespaces.

Reply:
I think `admin` will not be able to create `customresourcedefinitions`. That might be needed for manifestwork. I will add an additional clusterrole for crd together with admin.